extension/src/shared/layerContainers/hooks/useEvaluateGeneralAppearance.ts

```diff
@@ -429,13 +430,15 @@ export const useEvaluateGeneralAppearance = ({
     useFindComponent(componentAtoms ?? [], STYLE_CODE_FIELD),
   )?.preset?.code;
 
-  const appearanceObject = useMemo(() => getAppearanceObject(styleCodeString), [styleCodeString]);
+  const appearanceObject = useMemo(
```

Comment:
I prefer more specific name.
```suggestion
  const appearanceObjectForStyleCode = useMemo(
```

Reply:
Updated as `appearanceObjectFromStyleCode`. Thank you!